lib/perl/Genome/FeatureList/Command/Import.pm

```diff
@@ -208,9 +214,78 @@ sub _resolve_bed_file_from_directory {
         return $all_tracks[0];
     }
 
+    if ($self->_has_nimblegen_capture_primary_pair(@bed_files)) {
+        my $pair = $self->_nimblegen_capture_primary_pair(@bed_files);
+        return $self->_create_nimblegen_capture_primary_multitrack_bed_file($pair);
+    }
+
     die $self->error_message('Multiple candidate BED files found in directory %s. Please select one.', $directory);
 }
 
+sub _nimblegen_capture_primary_pair {
+    my $self = shift;
+    my @bed_files = @_;
+
+    my @suffixes = qw(_capture_targets.bed _primary_targets.bed);
+    my @matches;
+    for my $suffix (@suffixes) {
+        my @matching = grep { /\Q$suffix\E$/ } @bed_files;
+        my @prefixes = map { /(.*)\Q$suffix\E$/; $1 } @matching;
+        push @matches, Set::Scalar->new(@prefixes)
+    }
+    my $intersection = $matches[0]->intersection($matches[1]);
+
+    if ($intersection->is_empty) {
+        croak 'unable to find a capture/primary pair';
+    }
+
+    if ($intersection->size > 1) {
+        croak 'found multiple capture/primary pairs';
+    }
+
+
+    my $prefix = ($intersection->members)[0];
+    return {
+        capture => join('', $prefix, $suffixes[0]),
+        primary => join('', $prefix, $suffixes[1]),
+    };
+}
+
+sub _has_nimblegen_capture_primary_pair {
+    my $self = shift;
+    my @bed_files = @_;
+    my $pair = try { $self->_nimblegen_capture_primary_pair(@bed_files) };
+    return defined $pair;
+}
+
+sub _create_nimblegen_capture_primary_multitrack_bed_file {
+    my $self = shift;
+    my $pair = shift;
+
+    my @data = (
+        [Genome::Sys->open_file_for_reading($pair->{capture}), 'tiled_region',  (File::Spec->splitpath($pair->{primary}))[2]],
+        [Genome::Sys->open_file_for_reading($pair->{primary}), 'target_region', (File::Spec->splitpath($pair->{primary}))[2]],
+    );
+
+    my ($multitrack_fh, $multitrack_path) = Genome::Sys->create_temp_file();
+    for (@data) {
+        my ($fh, $name, $desc) = @{$_};
+        $multitrack_fh->printf(qq(track name=%s description="%s"\n), $name, $desc);
+        while (my $line = $fh->getline) {
+
+            # normalize chromosome names...
+            if (index($line, 'chr') == 0) {
+                $line = substr($line, 3);
```

Comment:
I don't think it's safe for us to do this here.  We have references that do contain `chr`s and this would cause the BED to fail verification against them.  (Most nimblegen BEDs are set to the reference returned by `genome model reference-sequence list-builds -f prefix=nimblegen`.

[If the two BEDs we're combining do come from different references, that's something that `Genome::FeatureList::Command::Merge` already tries to handle.]